Assets/FishNet/Runtime/Generated/Component/NetworkTransform/NetworkTransform.cs

```diff
@@ -451,6 +455,10 @@ internal void Update(uint tick, Vector3 position, Quaternion rotation, Vector3 s
         /// Number of intervals remaining before synchronization.
         /// </summary>
         private short _intervalsRemaining;
+        /// <summary>
+        /// Used to force resync. Acts the same as having Teleport enabled but is only active for one frame.
+        /// </summary>
+        private bool forceResync = false;

```

Comment:
You'll want to match the underscore prefix convention for private member variables.